walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

```diff
@@ -54,61 +51,157 @@ image::images/arch.png[architecture, role="integr8ly-img-responsive"]
 
 :sectnums:
 
-[time=10]
-== Deploying the Data Sync showcase application
+[time=15]
+== Creating your DataSync project from DataSync Starter template
 
-//Home > Search > Select Templates from the drop down > Search for datasync and they should show up. Just make sure you're on the openshift namespace
+{data-sync-name} allows you to focus on your business model by giving you ability
+to generate fully functional GraphQL based API and Node.js Server and Client Side components.
 
-. Go to the link:{openshift-host}/topology/ns/{walkthrough-namespace}[project console, window="_blank"] in OpenShift.
-. Select the *From Catalog* option.
-. Use the *Filter by keyword...* field to search the catalog for `Data Sync Showcase`.
-. Select the *Data Sync Showcase* option to add it to your project.
-. Click the *Instantiate Template* button.
-. Accept all the default values on the provisioning form, scroll down to the bottom of the screen, and click *Create*.
-. Go to link:{openshift-host}/k8s/ns/{walkthrough-namespace}/pods[Workloads > Pods, window="_blank"] page for your project. You should see the following three pods:
-+
-* *ionic-showcase-server* - a Node.js server and client application
-* *mosquitto-mqtt-broker* - a message broker
-* *postgresql* - a database
-+
-Wait until you see that all three pods have the *Running* status. Once the pods are running, you can use the GraphQL playground to interact with the server API as described in the next step.
+DataSync will allow your application to recieve live updates thanks to GraphQL subscriptions and
+operate independently of network connection. Developers can create GrapQL types as model
+and generate underlying backend that will work out of the box with the RHMI services like SSO or AMQ Online
+
+[time=30]
+=== Setting your DataSync project using DataSync Starter
+
+. Install yarn `npm install -g yarn`
+. Clone git repository: `git clone https://github.com/aerogear/datasync-starter#walkthrough`
```

Comment:
Could you do this with the master. I will push master to walkthrough once we happy with this WT